packages/db/src/models/base.ts

```diff
@@ -53,6 +53,15 @@ class BasePgModel<ItemType, RecordType extends { cumulus_id: number }> {
       .insert(item)
       .returning('cumulus_id');
   }
+
+  delete(
+    knexOrTransaction: Knex | Knex.Transaction,
+    params: Partial<RecordType>
+  ) {
+    return knexOrTransaction(this.tableName)
+      .where(params)
+      .del();
+  }
```

Comment:
I don't think you need to change this here but my PR will update this to be an async method so we can take advantage of promise rejections.